plugins/auth-backend/src/providers/okta/provider.ts

```diff
@@ -279,6 +282,10 @@ export const okta = createAuthProviderIntegration({
      * Looks up the user by matching their email local part to the entity name.
      */
     emailLocalPartMatchingUserEntityName: () => commonByEmailLocalPartResolver,
+    /**
+     * Looks up the user by matching their email to the entity email.
+     */
+    emailMatchingUserEntityEmail: () => commonByEmailResolver,
```

Comment:
```suggestion
    emailMatchingUserEntityProfileEmail: () => commonByEmailResolver,
```

to make this extra clear